code/datums/uplink_items/uplink_general.dm

```diff
@@ -525,6 +525,13 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/storage/box/syndie_kit/safecracking
 	cost = 5
 
+/datum/uplink_item/stealthy_tools/handheld_mirror
+	name = "Hand Held Mirror"
+	desc = "Change and color your hair on the go."
+	reference = "HM"
+	item = /obj/item/handheld_mirror
+	cost = 5
```

Comment:
Will need to be updated as per the big TC inflation PR that was just merged.

Reply:
That is it scaled up, I originally was putting for 1. I don't think 1 (5 now) is reasonable for the item potential power, but I'm not sure what a comfortable number would be for it atm